process/lifecycle.md

```diff
@@ -9,69 +8,205 @@ This lifecycle document is maintained by The Academy Software Foundation (“ASW
+
+Sandbox Projects may propose to be reviewed to move to the Incubation Stage at any time by adding the review to a future TAC meeting agenda ( minimum two weeks notice required ) or may be moved to the Incubation Stage during its annual review. Projects should prepare a presentation outlining how it has completed the Incubation Stage requirements.
+
+### Benefits
 
 Incubation stage projects are eligible to receive the following benefits:
 
-* Incubation stage projects will constitute “Technical Projects” under the ASWF charter and may receive support as determined by the Governing Board.
+* Be considered as one of the “Technical Projects” under the [ASWF Charter](https://github.com/AcademySoftwareFoundation/foundation/blob/master/CHARTER.md) and may receive support as determined by the Governing Board.
 * Neutral hosting of the project’s community and any key assets (e.g. trademark, domain, etc.)
 * Assistance from the ASWF TAC to facilitate collaboration with other project communities.
 * Blog announcement or similar communication announcing the inclusion of the project.
 * Right to refer to the project as an incubation project of ASWF, and an opportunity to participate in events and other collaborative activities sponsored by ASWF.
 * Subject to applicable trademark usage guidelines, to display ASWF’s logo on the project’s code repository.
 
-Incubation stage projects are expected to leverage third-party public code repositories.
+### Expectations
+
+Incubation Stage projects should provide a quarterly report to the TAC outlining its progress on completing the requirements for the Active Stage. 
+
+Every 12 months, each Incubation Stage project will be reviewed by the TAC to assess its progress towards graduating to the Active Stage. If the project has not met the requirements for graduating to the Active Stage, the TAC may renew the project at the Incubation Stage for another 12 months with a majority vote of the TAC. 
+
+It is expected that Incubation Stage projects graduate to the Adopted Stage within 2 years from moving to the Incubation Stage. In the case of an Incubation Stage project that is not renewed with ASWF, the trademark and any assets will be returned to the project maintainers or an organization they designate.
+
+## Adopted Stage
+
+Adopted Stage projects are considered mature projects that generally are ready for production use. Projects at this stage are focused on growing an ecosystem of users and are often being leveraged in vendor products or being used by end-users.
+
+### Requirements
+
+To be advance to an Adopted stage, a project must meet the Incubation stage requirements plus:
+
+* Demonstrate a substantial ongoing flow of commits and merged contributions, authored by a healthy number of diverse contributors*.
+* Demonstrable roadmap progress.
+* A healthy number of public adopters, identified within the project ( using an ADOPTERS.md page or having showcased on the project’s website )
```

Comment:
Trivial detail, but in previous sections we didn't specify that project files had to be in Markdown with a .md extension. Also this bullet item list has a mix of period and non period terminated items